rust/agents/checkpointer/src/submit.rs

```diff
@@ -50,7 +50,7 @@ impl CheckpointSubmitter {
                 );
                 // If there are any new messages, the count will be greater than
                 // the latest checkpoint index.
-                if count > latest_checkpoint_index {
+                if count > latest_checkpoint_index && count > 1 {
```

Comment:
Should this be `if count >= latest_checkpoint_index && count > 1 {`? If so also worth changing the comment

Reply:
ugh of course, it worked for the first time, since latest_checkpoint_index was 0 ...

Reply:
Oops yeah my bad